sim/src/analytics.rs

```diff
@@ -196,15 +196,15 @@ impl Analytics {
         if let Event::TripIntersectionDelay(trip_id, turn_id, delay) = ev {
             self.trip_intersection_delays
                 .entry(trip_id)
-                .or_insert_with(Vec::new)
-                .push((turn_id, delay));
+                .or_insert_with(|| BTreeMap::new())
```

Comment:
Small trick here, you can do `or_insert_with(BTreeMap::new)`, since the signature of `new` is also `Fn() -> BTreeMap`

Reply:
Ah that's a nice trick. Thanks